data/core/macros/utils.cfg

```diff
@@ -552,6 +552,9 @@
                         cumulative=no
                         affect_self=yes
                         overwrite_specials=both_sides
+                        [overwrite]
+                            priority=1000
```

Comment:
Is 1000 a special value, or is it just meant to be a really high value?

Reply:
It is a really high value